### ISSUE TITLE
TDDFT restart 

### ISSUE THREAD

**Issue by jjakowski** [RMGDFT/rmgdft#57]
There seems to be an issue with   restart  capabilities in TDDFT.  The dynamics seems to go smoothly  from the last point pf previous simulation when  restarted. However,  in the long term the trajectory ot  real-time TDDFT     seems altered and discontinuous. 

Below is an example of UVVIS dipole evolution for c60 when TDDFT was restarted after 1000 steps.  

![image](https://user-images.githubusercontent.com/6385536/75720567-c3f18a00-5ca4-11ea-953e-d41acc7cdba9.png)

 The red line corresponds to  previous simulations and the green one to restarted. Although at first everything looks smooth.  But  when the overall   time dependence of dipole is plotted then one can clearly see  that the dynamics has been altered.  and does not really follow the original, red curve trajectory:

![image](https://user-images.githubusercontent.com/6385536/75721029-90fbc600-5ca5-11ea-8cc4-49e770250641.png)

**Comment by jjakowski** [comment]
.. here is  the same  TDDFT simulations but the    initial TDDFT  goes uninterrupted past the time=1000.  The difference between  "red" and "green" curves indicates an altered trajectory due to  TDDFT restart:

![image](https://user-images.githubusercontent.com/6385536/75826250-17330d80-5d75-11ea-8ddc-3af1b3878c9f.png)

**Comment by jjakowski** [comment]
It turns out   that "potential acceleration" is large part responsible for  the TDDFT dynamics issues. It acts as a un-physical kick perturbation potential and alters trajectory of TDDFT dynamics.  Turning off potential acceleration (potential_acceleration_constant_step="0.0")  drastically improves the  quality of dynamics, as shown below  for C60:

![image](https://user-images.githubusercontent.com/6385536/76036756-f6082380-5f12-11ea-85ee-edd90c12fd06.png)


There is still however, some lingering issues with TDDFT restart. 
I did tests of  TDDFT restart reproducibility   with acceleration turned off. 

 Here is the plot for the same system:
![image](https://user-images.githubusercontent.com/6385536/76036171-657d1380-5f11-11ea-982c-730e223a1df8.png)

Here I made a checkpoint at  time=450  and then I used  it for a series of restarts (different colors). 
The green  one correspond to a single trajectory   starting from time=450 and ending at  time =775.
The red, blue and red are   correspond to  3 different  restarts done consecutive one after each other to mimic our  working scenario.  So basically  one can expect that  red-blue-red line  should follow exactly the  green line.  However, only then first red piece follows  the green line.  This suggests  that at the restart the TDDFT  is not exactly done the same way as during the normal run.  

Noticeably the amplitude of "blue" oscillations  (after the restart at time=500) is higher.
